Update vmw-how-convert-edge.md

mikepukcloud opened the pull request:
removed 8.20 bug and renamed the tab removing the 8.20 reference.

mikepukcloud commented:
Only reason I went with my names is “Flex” UI is not really the term that customers are used to, only since the upgrade to 9 have we been referring to that UI by its “flex” name.

Fonr tenant portal I don’t want them confused between our portal and then calling the VCD interface a “portal”, as its not really a portal, the term portal as defined in the oxford dictionary as “a website or web page providing access or links to other sites.” Which, as a UI for just vCloud, its not really a portal.

Agree that HTML5 isnt a great name either, so maybe “legacy UI” and “Tenant UI” might be better or “vCloud 9 UI”


Michael Perry - 
Principal Cloud Support Engineer
mperry@ukcloud.com
+44 7795 068 486
www.ukcloud.com
A8, Cody Technology Park, Ively Road, Farnborough, GU14 0LX
Notice: This message contains information that may be privileged or confidential and is the property of UKCloud Ltd. It is intended only for the person to whom it is addressed. If you are not the intended recipient, you are not authorised to read, print, retain, copy, disseminate, distribute, or use this message or any part thereof. If you receive this message in error, please notify the sender immediately and delete all copies of this message. UKCloud reserves the right to monitor all e-mail communications through its networks. UKCloud Ltd is registered in England and Wales: Company No: 07619797. Registered office: Hartham Park, Hartham, Corsham, Wiltshire SN13 0RP.
From: Sue Highmoor <notifications@github.com>
Sent: 03 May 2019 17:53
To: UKCloud/documentation <documentation@noreply.github.com>
Cc: Michael Perry <mperry@ukcloud.com>; Author <author@noreply.github.com>
Subject: Re: [UKCloud/documentation] Update vmw-how-convert-edge.md (#533)


@suehighmoor commented on this pull request.

I'm going through a process at the moment of identifying all the articles that refer to 8.20 and want to settle on a consistent way of referring to the two different UIs. I was going to go with Flex client for the legacy UI and tenant portal for the HTML5 UI. Any objections to that or specific reasons why you used the terminology you've suggested here?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/UKCloud/documentation/pull/533#pullrequestreview-233569827>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKLO3BLVJULLQSXAFJS3JTLPTRUYJANCNFSM4HKVB4FA>.

gellner commented:
> 
> 
> removed 8.20 bug and renamed the tab removing the 8.20 reference.

Should we remove the reference to 8.20 in the first paragraph?

The last paragraph would still mention 8.20 but it links to a useful document,

mikepukcloud commented:
I specifically didn’t remove it as I understand all documentation was being review for 9.7 to remove stuff like this, I wanted to jump the gun on this one as it should have been removed when we moved to 9.1 and I got asked twice from triage last week if it was still an issue as it was in the document and customers who had been asked to upgrade, had pointed it out

Happy to remove it though if all in agreement


Michael Perry - 
Principal Cloud Support Engineer
mperry@ukcloud.com
+44 7795 068 486
www.ukcloud.com
A8, Cody Technology Park, Ively Road, Farnborough, GU14 0LX
Notice: This message contains information that may be privileged or confidential and is the property of UKCloud Ltd. It is intended only for the person to whom it is addressed. If you are not the intended recipient, you are not authorised to read, print, retain, copy, disseminate, distribute, or use this message or any part thereof. If you receive this message in error, please notify the sender immediately and delete all copies of this message. UKCloud reserves the right to monitor all e-mail communications through its networks. UKCloud Ltd is registered in England and Wales: Company No: 07619797. Registered office: Hartham Park, Hartham, Corsham, Wiltshire SN13 0RP.
From: gellner <notifications@github.com>
Sent: 07 May 2019 12:42
To: UKCloud/documentation <documentation@noreply.github.com>
Cc: Michael Perry <mperry@ukcloud.com>; Author <author@noreply.github.com>
Subject: Re: [UKCloud/documentation] Update vmw-how-convert-edge.md (#533)


removed 8.20 bug and renamed the tab removing the 8.20 reference.

Should we remove the reference to 8.20 in the first paragraph?

The last paragraph would still mention 8.20 but it links to a useful document,

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/UKCloud/documentation/pull/533#issuecomment-490044382>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKLO3BMEEAQ74TSA3PXAZI3PUFTI7ANCNFSM4HKVB4FA>.
